plugins/woocommerce/src/Admin/Features/Features.php

```diff
@@ -164,8 +164,7 @@ public static function get_available_features() {
 			$feature_class = self::get_feature_class( $optional_feature_key );
 
 			if ( $feature_class ) {
-				$default = isset( self::$optional_features[ $optional_feature_key ]['default'] ) ?
-					self::$optional_features[ $optional_feature_key ]['default'] :
+				$default = self::$optional_features[ $optional_feature_key ]['default'] ??
 					'no';
```

Comment:
```suggestion
				$default = self::$optional_features[ $optional_feature_key ]['default'] ?? 'no';
```

It makes less sense to split this across multiple lines now that we have shortened the statement.